From 3d6bc9476055ced008d4e50b491c2e841d077011 Mon Sep 17 00:00:00 2001
From: Florian Schmaus <flo@geekplace.eu>
Date: Sun, 11 Apr 2021 11:10:23 +0200
Subject: [PATCH] Save HTML documentation in extra html/ directory

Some distributions, like Gentoo, have the policy that HTML
documentation should go under /usr/share/doc/<packet>/html. This
allows features like Gentoo's DOC_SYMLINKS_DIR.
---
 doc/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/CMakeLists.txt b/doc/CMakeLists.txt
index 7c52ffc8a345..2e94c99212c6 100644
--- a/doc/CMakeLists.txt
+++ b/doc/CMakeLists.txt
@@ -120,7 +120,7 @@ function(gen_html sourcefile)
             DEPENDS ${src} ${ARGN}
             )
     endif()
-    install(FILES ${dst} DESTINATION ${DOCDIR})
+    install(FILES ${dst} DESTINATION "${DOCDIR}/html")
 endfunction()
 
 
-- 
2.30.2

